src/main/drivers/accgyro_spi_mpu6000.c

```diff
@@ -119,8 +121,25 @@ bool mpu6000ReadRegister(uint8_t reg, uint8_t length, uint8_t *data)
     return true;
 }
 
-void mpu6000SpiGyroInit(uint8_t lpf)
+void mpu6000SpiGyroInit(gyro_t *gyro, uint8_t lpf)
 {
+    uint16_t intPeriod;
+    switch(lpf) {
+        case 0:
+            intPeriod = PERIOD_HZ(8000);
```

Comment:
The `PERIOD_HZ` really obfuscate the code here. So `intPeriod` is a time in microseconds, but instead of setting it using a time value, it is set using a period in Hz and a conversion macro. It's a bit like doing:

```
distanceInKm = MILES_TO_KM(5);
```

every time you look at the code, you have to do a mental calculation of to work out the actual value being set.